.github/workflows/release.yml

```diff
@@ -149,18 +153,24 @@ jobs:
         platforms: linux/amd64,linux/arm64/v8
         tags: ${{ steps.distroless-meta.outputs.tags }}
         push: true
-    - name: Build local distroless image, which is needed to run Snyk
-      if: matrix.run_snyk
+    - name: Publish local distroless image, which is needed to run Snyk
       run: sbt "project ${{ matrix.app }}Distroless" docker:publishLocal
     - name: Run Snyk to check for vulnerabilities
       uses: snyk/actions/docker@master
-      if: matrix.run_snyk
+      if: matrix.final_release
       with:
         image: "${{ steps.packageName.outputs.package_name }}:${{ steps.ver.outputs.tag }}-distroless"
         args: "--app-vulns --org=data-processing-new"
         command: monitor
       env:
         SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+    - name: Run Snyk to check for vulnerabilities (-rc)
+      uses: snyk/actions/docker@master
+      if: ${{ ! matrix.final_release }}
+      with:
+        image: "${{ steps.packageName.outputs.package_name }}:${{ steps.ver.outputs.tag }}-distroless"
+        args: "--app-vulns"
+        command: monitor
```

Comment:
Is the intention for this to go to our non-prod dashboard?

Reply:
Yeah, or just display it in the Github action, but that [failed](https://github.com/snowplow/enrich/actions/runs/7668099290/job/20899268003)

Reply:
How does it work though?  I assumed our `SNYK_TOKEN` is scoped only for one organization?  Or is that a wrong assumption -- maybe it has permissions to push to both organizations?

Should we specify the `--org=something` so we explicitly tell it where to send the results?

Reply:
You're right, it is scoped for `data-processing-new`. I removed it

`for queue in kinesis pubsub kafka nsq; do sbt "project ${queue}Distroless; set version := \"latest\"; Docker / publishLocal"; snyk container test --app-vulns snowplow/snowplow-enrich-$queue:latest-distroless; done` FTW!